src/frontend/apps/impress/src/components/quick-search/QuickSearchItemContent.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactNode } from 'react';
+
+import { useCunninghamTheme } from '@/cunningham';
+
+import { Box } from '../Box';
+
+export type QuickSearchItemContentProps = {
+  alwaysShowRight?: boolean;
+  left: ReactNode;
+  right?: ReactNode;
+};
+
+export const QuickSearchItemContent = ({
```

Comment:
Will you need it ?

Reply:
Yes i forgot to use it, i duplicate the right content logic inside DocSearchItem